lib/gruf/controllers/base.rb

```diff
@@ -71,8 +71,8 @@ def call(method_key, &block)
           send(method_key, &block)
         end
       rescue GRPC::BadStatus
-        raise # passthrough
-      rescue StandardError => e
+        raise # passthrough, to be caught by Gruf::Interceptors::Timer
+      rescue GRPC::Core::CallError, StandardError => e # CallError is not a StandardError
```

Comment:
Can we return `:grpc_call_error` as the second arg to the `fail!` method? Its not really an unknown error, it is a GRPC::CallError

Reply:
We would have to duplicate the error handling for CallError just for that, which I don't think is worth it. In this case, the error message that comes back should be sufficient to explain the error.